db/data/20241011081455_update_accredited_providers.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class UpdateAccreditedProviders < ActiveRecord::Migration[7.2]
+  def up
+    # C76 should be the accredited provider for
+    # https://www.publish-teacher-training-courses.service.gov.uk/publish/organisations/2DB/2024/courses
```

Comment:
Pointless comments, since it will, perhaps make `AccreditedProviderUpdater` reflect it instead.

Reply:
No these are only here to match up and explain what we're trying to do. 

These comments are extracted from the Trello card to avoid having to jump between 2 sources. 

The comments will be removed before merging